amplify/backend/function/subscribeToColonyPushNotifications/lib/index.ts

```diff
@@ -1,60 +1,47 @@
-const admin = require('firebase-admin');
-
-const { graphqlRequest } = require('./utils');
-const {
-  getProfile,
-  updateContributorNotificationSettings,
-} = require('./graphql');
+import * as admin from 'firebase-admin';
+import { Handler } from 'aws-lambda';
+
+import { graphqlRequest, notNull } from '../../utils';
+import { setEnvVariables } from '../../getParams';
+
+import {
+  TriggerEvent,
+  GetProfile_StcpnQuery,
+  GetProfile_StcpnQueryVariables,
+  GetProfile_StcpnDocument,
+  UpdateColonyContributorMutation,
+  UpdateColonyContributorMutationVariables,
+  UpdateColonyContributorDocument,
+} from './types';
 
 let apiKey = 'da2-fakeApiId123456';
 let graphqlURL = 'http://localhost:20002/graphql';
 let firebaseAdminConfig;
```

Comment:
Weren't these extracted into a util somewhere ? So that they don't need to be re-declared everywhere ?

Reply:
They are indeed (well the fallback for graphql queries is these local host varaibles) this code has been updated to reflect that